sbpy/calib/tests/test_sun.py

```diff
@@ -152,6 +156,7 @@ def test_kurucz_nan_error(self):
         NaNs in Kurucz file should not affect this calculation.
 
         """
+        pytest.importorskip("synphot")
         sun = Sun.from_builtin("Kurucz1993")
```

Comment:
The above codecov warning is very strange.

Reply:
It is because remote tests are not run as part of our PR CI testing.